randovania/games/fusion/logic_database/Sector 1 SRX.txt

```diff
@@ -427,6 +427,9 @@ Extra - room_id: [13]
   * Extra - door_idx: (29,)
   > Room Center
       Trivial
+  > Event - Stabilizer 3 Online
+      # Speed Boost the Stabilizer: https://youtu.be/V20W0F4dZP0
+      Speed Booster or Shinespark Tricks (Intermediate) or Disabled Entrance Rando
```

Comment:
these should all be`and`, plus this need level 0 locks, as well as the `After Stabilizer Southeast Online` event

Reply:
It seems that the stabilizer events are still internally labeled 1-5, do you happen to know which one pertains to southeast?

Reply:
the one that has the long name description of "south east"

Reply:
Actually, I found it in the DB. Southeast is 4

Reply:
for future reference, the stabilizers are labeled in the order you encounter them, assuming you do northeast first

Reply:
They are not labeled as such in the resource database:
![image](https://github.com/user-attachments/assets/94d2a933-964b-42bd-93a5-1f3edc13f372)

Reply:
oh huh